src/components/Button/CozyHome.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { translate } from 'cozy-ui/react/I18n'
+import { ButtonLink } from 'cozy-ui/react'
+
+const CozyHome = ({ t, from }) => (
+  <ButtonLink
+    label={t('Share.create-cozy')}
+    href={`https://cozy.io/try-it?from=${encodeURIComponent(from)}`}
+  />
+)
+
+CozyHome.propTypes = {
+  t: PropTypes.func.isRequired,
```

Comment:
I think you should get `t` from context rather than `props` or `props` should be at least optional with a default value in the button.